src/server/game/Entities/Unit/Unit.h

```diff
@@ -852,6 +852,7 @@ class TC_GAME_API Unit : public WorldObject
         bool IsTotem() const    { return (m_unitTypeMask & UNIT_MASK_TOTEM) != 0; }
         bool IsVehicle() const  { return (m_unitTypeMask & UNIT_MASK_VEHICLE) != 0; }
 
+        uint8 GetContentTuning() const { return m_unitData->ContentTuningID; }
```

Comment:
Must return a 32bit value (preferably same sign as m_unitData->ContentTuningID)

Reply:
Oops, I forgot to update the return type. Fixed